### PR TITLE
doc: translate shallow renderer

### PR DESCRIPTION
**Pull request by barrypeng6**
Keep the following words:

- shallow renderer
- shallow rendering -> shallow render
- child (component)
- assert

### CODE REVIEW

File: content/docs/addons-shallow-renderer.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ layout: docs
 category: Reference
 ---
 
-**Importing**
+**如何引入**
```
Comment (neighborhood999) [inline content/docs/addons-shallow-renderer.md]: I prefer `如何 import`

File: content/docs/addons-shallow-renderer.md [hunk 2]
```diff
@@ -15,9 +15,9 @@ var ShallowRenderer = require('react-test-renderer/shallow'); // ES5 with npm
 
 ## Overview {#overview}
```
Comment (neighborhood999) [inline content/docs/addons-shallow-renderer.md]:
```suggestion
## 概觀 {#overview}
```

File: content/docs/addons-shallow-renderer.md [hunk 3]
```diff
@@ -15,9 +15,9 @@ var ShallowRenderer = require('react-test-renderer/shallow'); // ES5 with npm
...
 
-When writing unit tests for React, shallow rendering can be helpful. Shallow rendering lets you render a component "one level deep" and assert facts about what its render method returns, without worrying about the behavior of child components, which are not instantiated or rendered. This does not require a DOM.
+在為 React 寫單元測試時，shallow rendering 十分有用。 Shallow rendering 可以只 render 「第一層」的 component，並且對 component 的 render 方法的返回值進行斷言，不必擔心 child component 的行為，child component並沒有被實例化或被 render。Shallow rendering 不依賴 DOM。
```
Comment (neighborhood999) [inline content/docs/addons-shallow-renderer.md]:
```suggestion
當在為 React 寫單元測試時，shallow render 十分有用。Shallow render 可以只 render 「第一層」的 component，並且對 component 的 render 方法的回傳值進行 assert，不必擔心 child component 的行為，child component 並沒有被實例化或被 render。Shallow render 不依賴 DOM。
```

File: content/docs/addons-shallow-renderer.md [hunk 4]
```diff
@@ -30,12 +30,12 @@ function MyComponent() {
 }
 ```
 
-Then you can assert:
+你可以使用斷言：
```
Comment (neighborhood999) [inline content/docs/addons-shallow-renderer.md]:
```suggestion
你可以使用 assert：
```

File: content/docs/addons-shallow-renderer.md [hunk 5]
```diff
@@ -30,12 +30,12 @@ function MyComponent() {
...
 
 ```javascript
 import ShallowRenderer from 'react-test-renderer/shallow';
 
-// in your test:
+// 測試程式:
```
Comment (neighborhood999) [inline content/docs/addons-shallow-renderer.md]: You don't need to translate comment, thank you.

File: content/docs/addons-shallow-renderer.md [hunk 6]
```diff
@@ -47,22 +47,22 @@ expect(result.props.children).toEqual([
 ]);
 ```
 
-Shallow testing currently has some limitations, namely not supporting refs.
+Shallow testing 目前有些限制，並不支援 refs。
 
-> Note:
+> 注意:
```
Comment (neighborhood999) [inline content/docs/addons-shallow-renderer.md]:
```suggestion
> 注意：
```

File: content/docs/addons-shallow-renderer.md [hunk 7]
```diff
@@ -47,22 +47,22 @@ expect(result.props.children).toEqual([
...
 >
-> We also recommend checking out Enzyme's [Shallow Rendering API](https://airbnb.io/enzyme/docs/api/shallow.html). It provides a nicer higher-level API over the same functionality.
+> 建議你可以看看 Enzyme [Shallow Rendering API](https://airbnb.io/enzyme/docs/api/shallow.html)。它在相同功能的基礎上提供更棒更高級的 API。
```
Comment (neighborhood999) [inline content/docs/addons-shallow-renderer.md]:
```suggestion
> 我們建議你可以查看 Enzyme 的 [Shallow Rendering API](https://airbnb.io/enzyme/docs/api/shallow.html)。它在相同功能的基礎上提供更棒更高階的 API。
```

File: content/docs/addons-shallow-renderer.md [hunk 8]
```diff
@@ -47,22 +47,22 @@ expect(result.props.children).toEqual([
...
 
 ## Reference {#reference}
```
Comment (neighborhood999) [inline content/docs/addons-shallow-renderer.md]:
```suggestion
## 參考 {#reference}
```

File: content/docs/addons-shallow-renderer.md [hunk 9]
```diff
@@ -47,22 +47,22 @@ expect(result.props.children).toEqual([
...
-You can think of the shallowRenderer as a "place" to render the component you're testing, and from which you can extract the component's output.
+你可以把 shallowRenderer 想成一個用來 render 測試中 component 的「空間」，並且可以從中提取該 component 輸出的內容。
 
-`shallowRenderer.render()` is similar to [`ReactDOM.render()`](/docs/react-dom.html#render) but it doesn't require DOM and only renders a single level deep. This means you can test components isolated from how their children are implemented.
+`shallowRenderer.render()` 類似於 [`ReactDOM.render()`](/docs/react-dom.html#render)，但它不依賴 DOM 且只 render 一層。這意味著你可以對測試的 component 及其 child component 進行隔離測試。
 
 ### `shallowRenderer.getRenderOutput()` {#shallowrenderergetrenderoutput}
 
-After `shallowRenderer.render()` has been called, you can use `shallowRenderer.getRenderOutput()` to get the shallowly rendered output.
+在 `shallowRenderer.render()` 被呼叫後，你可以使用 `shallowRenderer.getRenderOutput()` 來獲取該 component 第一層的輸出內容。
```
Comment (neighborhood999) [inline content/docs/addons-shallow-renderer.md]:
```suggestion
在 `shallowRenderer.render()` 被呼叫後，你可以使用 `shallowRenderer.getRenderOutput()` 來取得該 component 第一層的輸出內容。
```

File: content/docs/addons-shallow-renderer.md [hunk 10]
```diff
@@ -47,22 +47,22 @@ expect(result.props.children).toEqual([
...
 
-You can then begin to assert facts about the output.
+然後，就可以對輸出的內容進行斷言操作。
```
Comment (neighborhood999) [inline content/docs/addons-shallow-renderer.md]:
```suggestion
然後，就可以對輸出的內容進行 assert 操作。
```

File: content/docs/addons-shallow-renderer.md (hunk 5 above)
Comment (barrypeng6) [reply content/docs/addons-shallow-renderer.md]: ok.

File: content/docs/addons-shallow-renderer.md (hunk 1 above)
Comment (barrypeng6) [reply content/docs/addons-shallow-renderer.md]: ok

Comment (barrypeng6) [reply content/docs/addons-shallow-renderer.md]: done!

File: content/docs/addons-shallow-renderer.md (hunk 5 above)
Comment (barrypeng6) [reply content/docs/addons-shallow-renderer.md]: done

File: content/docs/addons-shallow-renderer.md [hunk 11]
```diff
@@ -6,18 +6,18 @@ layout: docs
 category: Reference
 ---
 
-**Importing**
+**如何 import**
```
Comment (neighborhood999) [inline content/docs/addons-shallow-renderer.md]: The **i**mport should be the capital letter. My apologies.

```suggestion
**如何 Import**
```

**Pull request merged**